libfreeipmi/include/freeipmi/spec/ipmi-ipv6-address-status-spec.h

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
OTOH, these are fields from the spec, and part of the spec, so I would think that they belong in libfreeipmi/include/freeipmi/spec/$mumble.  I settled for ipmi-lan-configuration-parameters-spec.h on the theory that it's the best fit of the existing files in the spec/ directory.